### ISSUE TITLE
[BUG] wine / windows apps, "\" is not displayed

### ISSUE THREAD

**Issue by cylgalad** [aristocratos/bpytop#313]
**Describe the bug**

When Wine / Windows apps are running, bpytop doesn't display any "\\" path separators for these (too) numerous processes.

**To Reproduce**

Launch any Wine / Windows app (for instance, run "winecfg"), check in bpytop by filtering "\\" (that at least works!).

**Expected behavior**

Bpytop should display all "\\" path separators correctly (like ps and htop do).

**Info (please complete the following information):**

- bpytop version: 1.0.65
- psutil version: 5.8.0
- Archlinux
- any terminal
- Font used: not font-related
- Python 3.9.5

**Comment by UmarJ** [comment]
Result of `cmdline()`:
```
['C:\\windows\\system32\\services.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\winedevice.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\plugplay.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\winedevice.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\winedevice.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\explorer.exe', '/desktop', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
['C:\\windows\\system32\\winecfg.exe', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '', '']
```

In bpytop:

![2021-05-26_15-03_1](https://user-images.githubusercontent.com/34779641/119641946-a3769800-be33-11eb-8134-b28106b4ba6d.png)


It's these `replace("\\", "")` calls causing this:

https://github.com/aristocratos/bpytop/blob/472185c0804548c67f1159bb9dfec44035b20004/bpytop.py#L3735
https://github.com/aristocratos/bpytop/blob/472185c0804548c67f1159bb9dfec44035b20004/bpytop.py#L3919

**Comment by aristocratos** [comment]
@cylgalad 
Should be fixable by changing the last `replace("\\", "")` to `replace("\\", "\\")`. Which should pick up any escaped `\` and replace it with a unescaped `\` without removing any non escaped `\`.

The reason it's needed is because any `\` escaped control character in the command line could mess up the text formatting/placement.

**Comment by UmarJ** [comment]
> @cylgalad
> Should be fixable by changing the last `replace("\\", "")` to `replace("\\", "\\")`. Which should pick up any escaped `\` and replace it with a unescaped `\` without removing any non escaped `\`.
> 
> The reason it's needed is because any `\` escaped control character in the command line could mess up the text formatting/placement.

What's the point of having a `replace("\\", "\\")`? Isn't it redundant?

**Comment by aristocratos** [comment]
@UmarJ 
Yeah, noticed that too, possible it was in bash that "\\" would pick up any escaped character...

Will need to remove the "\\" replace and add a couple of others, for example "\0" wouldn't show in the output but would add one character when testing the string with `len()` so would still mess with the output...

**Comment by UmarJ** [comment]
@aristocratos
The strings returned by `cmdline()` already have the `\` character escaped.

For example, if I run `python sleep.py '\t\n\0\\'`, `cmdline()` returns `['python', 'sleep.py', '\\t\\n\\0\\\\']`. If I `print` the last element in the list, I get `\t\n\0\\` and `len('\\t\\n\\0\\\\')` correctly returns `8`.

So there shouldn't be a problem with escaped characters here, and we can remove all `replace()` calls in the two lines above, including the ones replacing `\n` and `\t`.

**Comment by aristocratos** [comment]
@UmarJ 
You're right. It was a issue in `bashtop` so I guess I added it as a precaution without actually looking in to it properly.

**Comment by aristocratos** [comment]
@cylgalad 
Fixed in v1.0.67
